src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -8,45 +8,47 @@
 
 
 public class GcpClient {
-    public static  String query;
+
     public static int noOfRecords;
 
-    public static int countBqQuery(String table) throws IOException, InterruptedException {
+    public  int countBqQuery(String table) throws IOException, InterruptedException {
 
         String projectId = SeleniumHelper.readParameters("Project-ID");
         String datasetName = SeleniumHelper.readParameters("Data-Set");
-         query = "SELECT count(*) "
+        String selectQuery = "SELECT count(*) "
                 + " FROM `"
                 + projectId
                 + "."
                 + datasetName
                 + "."
                 + table
                 + "`";
-         createBigQueryClient();
+         createBigQueryClient(selectQuery);
         return noOfRecords;
     }
     //Deleting the table
-    public static void dropBqQuery(String table) throws IOException, InterruptedException {
+    public  void dropBqQuery(String table) throws IOException, InterruptedException {
 
         String projectId = SeleniumHelper.readParameters("Project-ID");
         String datasetName = SeleniumHelper.readParameters("Data-Set");
-        String query = "DROP TABLE `"
+        String dropQuery = "DROP TABLE `"
                 + projectId
                 + "."
                 + datasetName
                 + "."
                 + table
                 + "`";
-        createBigQueryClient();
+        createBigQueryClient(dropQuery);
     }
 
-    public static void createBigQueryClient() throws InterruptedException, IOException {
+    public  void createBigQueryClient(String query) throws InterruptedException, IOException {
```

Comment:
As soon as you are not only creating a client, but also executing a query, name the method appropriately (e.g. `executeQuery`).